src/demos/Modals.vue

```diff
@@ -384,5 +386,9 @@ import Button from "./../components/Button.vue";
     Button
   }
 })
-export default class Modals extends Vue {}
+export default class Modals extends Vue {
+  logClick(string: string){
+    console.log(string);
```

Comment:
can we remove this log?

Reply:
Sure, I can remove it. It's purpose was to log and check in the Demos if the modal events were firing 1x instead of 2x again.